packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import styled from "styled-components";
+import { Wheelchair } from "@styled-icons/foundation/Wheelchair";
+
+interface WrapperProps {
+  color: string;
+  large: boolean;
+}
+
+const Wrapper = styled.div<WrapperProps>`
+  display: flex;
+  align-items: center;
+  background-color: ${props => props.color};
+  border-radius: ${props => (props.large ? "4px" : "20px")};
+  padding: 0.25em 0.6em 0.25em 0.4em;
+  margin-top: 0.25em;
+  word-wrap: anywhere; /* this can often look quite bad, but helps encourage icons */
+  max-height: 50px;
+
+  width: ${props => (props.large ? "65px" : "50px")};
+  justify-content: space-between;
+`;
+const StatusWrapper = styled.span`
+  flex: 1;
+
+  span {
+    display: block;
+  }
+`;
+
+const AccessibilityRating = ({
+  gradationMap,
```

Comment:
I meant https://www.merriam-webster.com/dictionary/gradation but it might be a bit confusing yeah. Could change it but would have to happen across a few PRs so we should only do it if we have a much much better name!